packages/types/src/sdk/row.ts

```diff
@@ -19,6 +25,7 @@ export interface SearchParams {
   indexer?: () => Promise<any>
   rows?: Row[]
   countRows?: boolean
+  aggregations?: Aggregation[]
```

Comment:
This is the API that takes us from the controller level into the SDK level. Everything below the SDK level has to interrogate this field and change behaviour based on it.